client/src/utils/auth-token.js

```diff
@@ -21,3 +21,7 @@ export const getAuthToken = () => {
 
   return token;
 };
+
+export const removeAuthToken = () => {
+  localStorage.removeItem(KEY_TOKEN);
```

Comment:
@agmaal1 looks good except that we shouldn't use local storage for the JWT token by default
this was done in first version, but now there is a github issue about that
might be out of scope here but if we use a storage for the token it should probably be session storage if the user doesn't choose to stay logged in

also see #656 and my comment